CMakeLists.txt

```diff
@@ -134,11 +144,14 @@ add_subdirectory(gui)
 if(WIN32)
   message("Going to gather dependencies on Windows")
   install(CODE "set(PLUGINS \"${PLUGINS}\")")
+  if (SCROOMBINDIR)
+    install(CODE "set(SCROOMBINDIR \"${SCROOMBINDIR}\")")
+  endif()
   install(
     CODE [[
       message("Gathering dependencies for: ${PLUGINS}")
       file(GET_RUNTIME_DEPENDENCIES
-        EXECUTABLES bin/scroom.exe
+        EXECUTABLES ${SCROOMBINDIR}bin/scroom.exe
```

Comment:
Add if statement around this, to clarify that SCROOMBINDIR is an optional variable.  (I believe that non existant CMake variables are rendered as empty strings, so functionally they are same)